### ISSUE TITLE
[GitHub] Bug: Update `[p]github whatlinks` Instructions

### ISSUE THREAD

**Issue by TwinDragon** [Obi-Wan3/OB13-Cogs#10]
Not a bug per say, but something I noticed while playing around with it and it's not a pressing matter.

In the `github whatlinks` description for a private repository, it says to search for the `atom` link in the element inspection for the browser. It took a bit of digging around, but I think it would be good to note that in order to see this atom link, you'd need to view the `Commit History` for a particular branch.

When I saw in #2 about the different branches, I believe simply just going to the commit history for a selected branch would suffice to get those specific commits. Looks like the basic setup is this:

```
https://github.com/<Username>/<repository>/commits/<branch>.atom?token=<token>
```

This should help a bit to try and make things a bit clearer, possibly even to the end-user who wants to use the cog to set it up on their own. As it stands currently, going to the repo and searching in the element inspection for something in regards to `atom` doesn't return any results, which lead me to search around to find something in the actual commit history.

**Comment by Obi-Wan3** [comment]
@TwinDragon Thanks for bringing this to my attention, when I first created this cog a few months ago (it was private for quite a bit before I started my repo), those instructions worked and the `.atom` file could be found in the root repo link. I just checked myself too and it has indeed moved; however, I am hesitant about correcting it immediately as I haven't looked at this cog for a while, and am unsure of how it would handle different branches, etc. which this may affect if users go to a different branch's `.atom`. That's one of the updates I am working on, if I get more time before then to look then I will correct this beforehand but am too busy right now (hopefully in the next week or two).
